website/static/css/pages/share-buttons.css

```diff
@@ -0,0 +1,37 @@
+.share-buttons {
+    margin: 3px 20px;
+}
+
+.share-buttons a {
+    padding: 0px 2px;
+    display: inline-block;
+}
+
+.share-buttons .fa {
+    color: #08c;
+    font-size: 1.6em;
+    line-height: 1.6em;
+}
+
+#share .share-buttons {
```

Comment:
Ah ok. Even so, we try to avoid using CSS IDs in selectors because they aren't reusable. Is it necessary to include `#share` in the selector?